src/main/java/com/commercetools/sync/products/utils/ProductSyncUtils.java

```diff
@@ -0,0 +1,4 @@
+package com.commercetools.sync.products.utils;
+
+public interface ProductSyncUtils {
```

Comment:
It is obviously not intended. The product sync (all classes in the package sync/products) isn't implemented yet. I deleted them to avoid confusion in the pr: b2bed41